src/components/cookie-consent-drawer.less

```diff
@@ -1,15 +1,55 @@
-.CookieConsentDrawer-contentContainer {
+.CookieConsentDrawer {
+  display: none;
+
+  &.is-enabled {
+    display: block;
+  }
+}
+
+.CookieConsentDrawer-overlay {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 1000;
+  background-color: rgba(0, 0, 0, 0.45);
+}
+
+.CookieConsentDrawer-drawer {
+  background: fade(@global-warning-color, 85%);
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+  position: fixed;
+  left: 0;
+  bottom: 0;
   width: 100%;
-  display: flex; 
+  z-index: 1001;
+}
+
+.CookieConsentDrawer-icon {
+  flex: none;
+  display: flex;
+  align-items: center;
   justify-content: center;
+  padding: 15px 25px;
```

Comment:
Maybe this could be `padding: @cookie-consent-drawerr-padding;`

Reply:
True, extracted, thanks